### ISSUE TITLE
DeprecationWarning: 'werkzeug.contrib.atom' is deprecated as of version 0.15 and will be removed in version 1.0

### ISSUE THREAD

**Issue by webknjaz** [lektor/lektor-atom#32]
I've just noticed this in my dev server run:
```console
$ python -m lektor server --host 0.0.0.0 --port 5000 -v
~/.cache/lektor/packages/423d074e94325a61c404d144348525b7/lektor_atom.py:19: DeprecationWarning: 'werkzeug.contrib.atom' is deprecated as of version 0.15 and will be removed in version 1.0.
  from werkzeug.contrib.atom import AtomFeed
```

Apparently, `werkzeug.contrib.atom` is going away and has to be replaced by some alternative library.

**Comment by runfalk** [comment]
This is known and is actually already fixed on master. We do however need to make a new release.

**Comment by dwt** [comment]
I'd like to second that - is there anything I can do to help make a release?

**Comment by dwt** [comment]
@maintainers? ping?

**Comment by webknjaz** [comment]
@goanpeca? @runfalk? @xlotlu? Do you need any help fixing an automated release process? Any blockers?

**Comment by webknjaz** [comment]
@dwt FYI according to https://www.getlektor.com/docs/plugins/#loading-plugins it's possible to just drop-in the source into the `plugins/` dir in your project as a workaround.

**Comment by dwt** [comment]
@webknjaz Not sure where you found that on those pages. I have installed the plugins via the 'packages' folder of my project, which actually raised the issue that the `extra_requires` of this projects doesn't jive with the way lektor installs the plugin.

**Comment by webknjaz** [comment]
@dwt did you follow the dev guide at https://www.getlektor.com/docs/plugins/dev/? My understanding is that you should be able to clone the entire repo under `plugins/` and enable it in the config...

**Comment by webknjaz** [comment]
It looks like [`lektor dev new-plugin PLUGIN_NAME`] can generate you the right structure which you could fill out...

[`lektor dev new-plugin PLUGIN_NAME`]: https://www.getlektor.com/docs/cli/dev/new-plugin/.

**Comment by xlotlu** [comment]
@webknjaz sorry, I'm not a maintainer anymore. I have no idea why this isn't fixed yet.

**Comment by dwt** [comment]
> @dwt did you follow the dev guide at https://www.getlektor.com/docs/plugins/dev/? My understanding is that you should be able to clone the entire repo under `plugins/` and enable it in the config...

I think I get it, the documentation mentioned the 'package' directory (which I already am using), which is also what `lektor dev new-plugin` recommends. So I guess that is what you meant?

Base result: I can install the plugin - if II change it to comment the `extra-requires` from the setup.py file. This is likely requiring a fix from lektor itself, as the way they install local plugins doesn't work with `extra-requires` in the setup.py file.

**Comment by dwt** [comment]
Installing from pypi might work better, so that's why I asked for a release.

**Comment by webknjaz** [comment]
> Installing from pypi might work better, so that's why I asked for a release.

No objections here. I was just trying to be helpful because it doesn't look like it's going to be published anytime soon.



> Base result: I can install the plugin - if II change it to comment the `extra-requires` from the setup.py file. This is likely requiring a fix from lektor itself, as the way they install local plugins doesn't work with `extra-requires` in the setup.py file.

If it's just `extras_require`, it should be safe to remove as it's unnecessary in runtime: this package only has an extra for tests. With `entry_points` it'd be difficult, though.

**Comment by dwt** [comment]
@webknjaz do you know who can cut a release for this package? Who do we need to talk to?

**Comment by webknjaz** [comment]
No idea, I tagged a few folks who made the last commits to master but besides that I don't feel like this plugin is maintained actively.

Maybe @yagebu knows...

**Comment by dwt** [comment]
Lets hope my pull request in lektor gets merged soon - then that would make a release here less neccessary. Thou still desirable.

**Comment by yagebu** [comment]
I updated the CI scripts (#37), so now it only takes a tag to make a new release - and I've just released v0.4.0 :)
